<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeDisplay</class>
 <widget class="QWidget" name="TimeDisplay">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_hourGroup">
     <property name="title">
      <string comment="@title:column">Hours</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="DigitDisplay" name="m_hourDisplay" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_minGroup">
     <property name="title">
      <string comment="@title:column">Minutes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="DigitDisplay" name="m_minDisplay" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_secGroup">
     <property name="title">
      <string comment="@title:column">Seconds</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="DigitDisplay" name="m_secDisplay" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_fracGroup">
     <property name="title">
      <string comment="@title:column">Hundredths</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="DigitDisplay" name="m_fracDisplay" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DigitDisplay</class>
   <extends>QWidget</extends>
   <header>digitdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
